recipes/parallel-meta-suite/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+mkdir -p ${PREFIX}/bin
+
+make CC=$CXX 
+
+cp -r bin/* ${PREFIX}/bin
+
+# # path for database and example
+ParallelMETA_path=${PREFIX}/${PKG_NAME}-${PKG_VERSION}
+mkdir -p ${ParallelMETA_path}
+
+# set VIBRANT DB PATH variable on env activation
```

Comment:
This comment isn’t quite correct, or is it? You can just delete that line, it’s clear what an activate/deactivate script does.

Reply:
The comment is correct. The database which we used is very big, more than 700MB. I don't think add this to the `source` is a good choice. So I define a path in the activate script, the download command will download the database in this folder.

Reply:
I just found it confusing that the comment seems to imply that a variable named `VIBRANT_DB_PATH` or similar would be set, but the actual variable name is `ParallelMETA_path`

Reply:
I’ll just merge it